src/Microsoft.AspNetCore.Server.Testing/Deployers/ApplicationDeployer.cs

```diff
@@ -79,7 +79,7 @@ protected void DotnetPublish(string publishRoot = null)
             DeploymentParameters.ApplicationPath =
                 (DeploymentParameters.ServerType == ServerType.IISExpress ||
                  DeploymentParameters.ServerType == ServerType.IIS) ?
-                Path.Combine(DeploymentParameters.PublishedApplicationRootPath, "wwwroot") :
+                DeploymentParameters.PublishedApplicationRootPath :
```

Comment:
I think that was intentional. The line I modified lines up with the tabs. The line above it aligns with the boolean expression.